# @file:	throw_error.py
# @author:	Jacob Xie
# @date:	2023/06/10 20:03:20 Saturday
# @brief:

import datetime as dt

raise Exception(f"Throw error at {dt.datetime.now()}")
